### ISSUE TITLE
False positive for dynamically constructed keys

### ISSUE THREAD

**Issue by timbru31** [seveves/ngx-translate-zombies#2]
Hi there 👋,

first of all: thanks for this project, really nice to find zombies so easily.

However I've ran into some false positives. I am sure this isn't an easy issue to solve, but wanted to report it nevertheless.

Example code snippets:
```html
<p class="error">{{'LOGIN.ERROR.' + (loginCode | async) | translate}}</p>
```
(Where `loginCode` is a string from a selector.)

Another example:
```ts
	getFilterName(filter: WalksheetFilter) {
		const name = WalksheetFilterType[filter.type];
		return this.translate.instant(`WALKSHEET.ACTIONBAR.FILTER.${name}`);
	}
```

ngx-translate-zombies marks these keys (like `LOGIN.ERROR.NOT_AUTHORIZED`) as a zombie, but they are definitively used via the dynamic key construction.

Any ideas how to solve this?

**Comment by seveves** [comment]
yeah the extension only works with dot-separated strings at the moment.
Maybe we can mark dynamic keys (keys where we are not sure if they are really in use anymore) in some way in the diff.
